bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -0,0 +1,458 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages, injectIntl } from 'react-intl';
+import Dropzone from 'react-dropzone';
+import update from 'immutability-helper';
+import cx from 'classnames';
+
+import ModalFullscreen from '/imports/ui/components/modal/fullscreen/component';
+import Icon from '/imports/ui/components/icon/component';
+import ButtonBase from '/imports/ui/components/button/base/component';
+import Checkbox from '/imports/ui/components/checkbox/component';
+import styles from './styles.scss';
+
+const propTypes = {
+  defaultFileName: PropTypes.string.isRequired,
+  fileSizeMin: PropTypes.number.isRequired,
+  fileSizeMax: PropTypes.number.isRequired,
+  handleSave: PropTypes.func.isRequired,
+  fileValidMimeTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+  presentations: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    filename: PropTypes.string.isRequired,
+    isCurrent: PropTypes.bool.isRequired,
+    conversion: PropTypes.object,
+    upload: PropTypes.object,
+  })).isRequired,
+};
+
+const defaultProps = {
+  defaultFileName: 'default.pdf',
+};
+
+const intlMessages = defineMessages({
+  title: {
+    id: 'app.presentationUploder.title',
+    defaultMessage: 'Presentation',
+  },
+  message: {
+    id: 'app.presentationUploder.message',
+    defaultMessage: `As a presenter in BigBlueButton, you have the ability of
+     uploading any office document or PDF file. We recommend for the best results,
+     to please upload a PDF file.`,
+  },
+  confirmLabel: {
+    id: 'app.presentationUploder.confirmLabel',
+    defaultMessage: 'Start',
+  },
+  confirmDesc: {
+    id: 'app.presentationUploder.confirmDesc',
+    defaultMessage: 'Save your changes and start the presentation',
+  },
+  dismissLabel: {
+    id: 'app.presentationUploder.dismissLabel',
+    defaultMessage: 'Cancel',
+  },
+  dismissDesc: {
+    id: 'app.presentationUploder.dismissDesc',
+    defaultMessage: 'Closes and discarts your changes',
+  },
+  dropzoneLabel: {
+    id: 'app.presentationUploder.dropzoneLabel',
+    defaultMessage: 'Drag files here to upload',
+  },
+  browseFilesLabel: {
+    id: 'app.presentationUploder.browseFilesLabel',
+    defaultMessage: 'or browse for files',
```

Comment:
Maybe just attach this part to the dropzoneLabel.defaultMessage?
This is just half of the message,

Reply:
Each phrase has a different style. Didn't find a better way to do it without two messages, any ideas?